app/controllers/application_controller.rb

```diff
@@ -17,4 +17,12 @@ def switch_language
   def set_locale
     I18n.locale = session[:locale] || I18n.default_locale
   end
+
+  def logged_in_user
+    return if logged_in?
+
+    flash[:danger] = I18n.t("controller.user.not_login")
```

Comment:
create locales/controllers to use this message
```suggestion
    application:
         user:  
            not_login: "User need require login"
         
```